src/test/script_tests.cpp

```diff
@@ -2623,12 +2623,170 @@ BOOST_AUTO_TEST_CASE(IsWitnessProgram)
     }
 }
 
+
+class AlwaysGoodSignatureChecker : public BaseSignatureChecker
+{
+protected:
+    unsigned int nFlags = SCRIPT_ENABLE_SIGHASH_FORKID;
+
+public:
+    //! Verifies a signature given the pubkey, signature and sighash
+    virtual bool VerifySignature(const std::vector<uint8_t> &vchSig,
+        const CPubKey &vchPubKey,
+        const uint256 &sighash) const
+    {
+        if (vchSig.size() > 0)
+            return true;
+        return false;
+    }
+
+    //! Verifies a signature given the pubkey, signature, script, and transaction (member var)
+    virtual bool CheckSig(const std::vector<unsigned char> &scriptSig,
+        const std::vector<unsigned char> &vchPubKey,
+        const CScript &scriptCode) const
+    {
+        if (scriptSig.size() > 0)
+            return true;
+        return false;
+    }
+
+    virtual bool CheckLockTime(const CScriptNum &nLockTime) const { return true; }
+    virtual bool CheckSequence(const CScriptNum &nSequence) const { return true; }
+    virtual ~AlwaysGoodSignatureChecker() {}
+};
+
+uint evalForSigops(const CScript &scriptSig,
```

Comment:
`uint`?

Reply:
interesting... not available in windows.  Will fix